portfolio/src/main/webapp/script.js

```diff
@@ -95,6 +164,34 @@ function createMap() {
   typeOfSearch('type-establishment', ['establishment']);
 }
 
+//prints all element on the arraylist
+function getReports(){
+  fetch('/numberOfReports')
+  .then(response => response.json())
+  .then((numReports) => {
+    numReports.forEach((grid) => {
+      console.log("NumberOfReports of grid: ");
+      console.log(grid.numOfReports);
+    });
+   })
+  .catch((error) => {
+    console.error(error);
+  });
+}
+
+//prints only the number of reports of a specific grid
+/*function getReportsForGrid(){
+  fetch("/numberOfReports?row=3&col=11")
```

Comment:
this is what we want. just make sure the servlet works with some hard coded values